mon-pix/tests/integration/components/comparison-window_test.js

```diff
@@ -78,15 +78,15 @@ describe('Integration | Component | comparison-window', function () {
       expect(find('.comparison-window-content-body__instruction')).to.exist;
     });
 
-    it('should render a feedback panel already opened', async function () {
+    it('should render a closed feedback panel', async function () {
       //when
       await render(
         hbs`<ComparisonWindow @answer={{this.answer}} @closeComparisonWindow={{this.closeComparisonWindow}} />`
       );
 
       //then
       expect(find('.comparison-window__feedback-panel')).to.exist;
-      expect(find('.feedback-panel__form')).to.exist;
+      expect(find('.feedback-panel__open-button')).to.exist;
```

Comment:
Cette classe existe dans tous les cas, que le panel soit fermé ou ouvert.
Est-ce que tu voulais juste tester qu'il soit présent à l'écran ?

Aussi, est-ce que ça vaut pas le coup de tester que `expect(find('.feedback-panel__form')).to.not.exist;` ?

Reply:
Yes je suis d'accord !